org.dpsoftware.FireflyLuciferin.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Firefly Luciferin
+Comment=Ambilight clone
+Exec=/app/fireflyluciferin/bin/FireflyLuciferin
```

Comment:
NEVER an absolute path. The executable should be installed in the standard `PATH` (which is `/app/bin` in that case)

Reply:
fixed it, thanks.